test/SIL.XForge.Scripture.Tests/Services/ParatextSyncRunnerTests.cs

```diff
@@ -1011,7 +1011,7 @@ public async Task SyncAsync_ParatextMissingAllChapters()
         [Test]
         public async Task RunAsync_NoRecordOfSyncedToRepositoryVersion_DoesFullSync()
         {
-            foreach (bool isChanged in new bool[] { true, false })
+            foreach (bool _ in new bool[] { true, false })
```

Comment:
I'm confused what's going on here. It looks like this is looping to run the test twice, presumably to test what happens the second time through. But a new TestEnvironment is created, so I would think the second run through would have lost the context of the first, and therefore be testing nothing?

Reply:
Hmmm you are right. I originally thought it needed to run twice, but it looks like the environment around the test changed, but the test did not keep up. I have removed the foreach loop.